tests/test_pylast.py

```diff
@@ -1909,54 +1908,4 @@ def test_get_recent_tracks_from_to(self):
         self.assertEqual(str(tracks[0].track.artist), "Johnny Cash")
         self.assertEqual(str(tracks[0].track.title), "Ring of Fire")
 
-
-if __name__ == '__main__':
```

Comment:
OK, great! Most of main can go then, I'd to keep just this basic call so it can still be run without py.test:

```python
if __name__ == '__main__':
    unittest.main(failfast=True)
```